hawc/apps/assessment/views.py

```diff
@@ -666,9 +666,26 @@ def isTruthy(self, request, field):
     def post(self, request, *args, **kwargs):
         if not request.is_ajax():
             return HttpResponseNotAllowed(["POST"])
+        response = {}
         if request.POST.get("hideSidebar"):
-            request.session["hideSidebar"] = self.isTruthy(request, "hideSidebar")
-        return HttpResponse(True)
+            hide_status = self.isTruthy(request, "hideSidebar")
+            request.session["hideSidebar"] = hide_status
+            response = {"hideSidebar": hide_status}
+        if request.POST.get("refresh"):
+            if request.user.is_authenticated:
+                old_time = request.session.get_expiry_date().isoformat()
+                request.session.set_expiry(None)  # use the global session expiry policy
+                new_time = request.session.get_expiry_date().isoformat()
+                response = {
+                    "message": f"Session extended from {old_time} to {new_time}.",
+                    "new_expiry_time": new_time,
+                }
+            else:
+                response = {
```

Comment:
we hit this case if a request isn't bound to a user, which happens when a session expires